<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Annotator</class>
 <widget class="QWidget" name="Annotator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Annotator.ico</normaloff>Annotator.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QFrame" name="labelsFrame">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="21" column="0">
          <widget class="QPushButton" name="addDelete">
           <property name="text">
            <string>(Add / Remove) Delete Set</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QPushButton" name="componentsButton">
           <property name="text">
            <string>Components</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="undoAll">
           <property name="text">
            <string>Undo All</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QPushButton" name="rotateLeft">
           <property name="text">
            <string>Rotate Left</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="drawWireframe">
           <property name="text">
            <string>Draw wireframe</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QPushButton" name="beginAnnotateButton">
           <property name="text">
            <string>Annotate...</string>
           </property>
          </widget>
         </item>
         <item row="25" column="0">
          <widget class="QPushButton" name="generateThumbnails">
           <property name="text">
            <string>Generate Thumbnails</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QTreeWidget" name="labelsTreeView">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="13" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="23" column="0">
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="curPage">
           <property name="text">
            <string>Page</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="selectedMesh">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="showGrids">
           <property name="text">
            <string>Show Grids</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="undoButton">
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QPushButton" name="rotateRight">
           <property name="text">
            <string>Rotate Right</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="22" column="0">
          <widget class="QListWidget" name="deleteList">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="curLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="noneButton">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QPushButton" name="remainButton">
           <property name="text">
            <string>Remaining</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="coarseLabels">
           <property name="text">
            <string>Show coarse labels</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="viewersFrame">
        <property name="styleSheet">
         <string notr="true">background: rgb(35, 35, 35);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="Line" name="line">
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QScrollBar" name="viewersScroll">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
